dao/GlossaryDAO.php

```diff
@@ -0,0 +1,109 @@
+<?php
+namespace Glossary\DAO;
+
+class GlossaryDAO {
+        
+    private $PDOX;
+    private $p;
+
+    public function __construct($PDOX, $p) {
+        $this->PDOX = $PDOX;
+    }
+   
+    // getAllDomains
+    // returns glossary_domains
+    function getDomain($domain_id = 0, $limit = 0, $offset = 0) {   
+        $arr = array();
+        $query = "SELECT *
+                    FROM `{$this->p}glossary_domain` `domain`";
+                    
+        if ($domain_id > 0) {
+            $query .= "WHERE `domain`.`id`=:domain_id";
+            $arr = array(':domain_id' => $domain_id);
+        }
+
+        return $this->PDOX->allRowsDie($query, $arr);
+    }
+    function getAlphabetTerms($link_id, $alphabet) {    
+         $query = "SELECT `id`, `domain_id`, `term`, `description`  
+                    FROM `{$this->p}glossary_term` 
+                    WHERE  `term` LIKE :alphabetPrefix";     
+                    $arr = array(   ':alphabetPrefix' => $alphabet . '%'    );     return $this->PDOX->allRowsDie($query, $arr); }
```

Comment:
Spacing:
- spaces around array value
- return should be on a new line

Line between functions